modules/transactions/src/main/java/org/apache/ignite/internal/tx/TxManager.java

```diff
@@ -155,6 +156,22 @@ CompletableFuture<Void> cleanup(
             UUID txId
     );
 
+    /**
+     * Sends cleanup request to the cluster nodes that hosts primary replicas for the enlisted partitions.
+     *
+     * @param enlistedPartitions Enlisted partition groups.
```

Comment:
It would be nice to explain through javadoc, why sometimes it's just TableParitionIds and sometimes Map<TablePartitionId, String>.